### PR TITLE
Newoptionsquoteplugin

### PR DESCRIPTION
**Pull request by aaronedell**
Get an API key and put it here (sign up for one for free at https://www.yahoofinanceapi.com). Input some variables in the file. Then ```go build``` the file in the plugins folder and rename.

### CODE REVIEW

File: Finance/001-options.1h.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
+//usr/bin/env go run $0 $@; exit
+//<xbar.title>Options Call Prices</xbar.title>
+//<xbar.version>1.0</xbar.version>
+//<xbar.author>Aaron Edell</xbar.author>
+//<xbar.author.github>aaronedell</xbar.author.github>
+//<xbar.desc>Track Specific Option Call Quotes</xbar.desc>
+//<xbar.image>https://d33wubrfki0l68.cloudfront.net/ea583c177bf610b4db04246ef394d583201feae7/870d6/assets/img/logo.png</xbar.image>
```
Comment (matryer) [inline Finance/001-options.1h.go:7]: Hey @aaronedell congrats on this plugin - can you provide an image of the plugin in use?

Comment (aaronedell) [reply Finance/001-options.1h.go]: I've committed the change to my branch. Do I have to do another pull request?

**Comment by sprak3000** [comment]
@aaronedell 

I tried running this locally on my M1 silicon machine. Got a shebang error:

Screenshot 2023-01-23 at 20 13 03

Tried fixing the shebang to `#! //usr/bin/env go run $0 $@; exit` but got a "Go not found" error. Might be something misconfigured on my machine though. Any thoughts on what install step I might need to get Go working properly for it?

Also, the script needs to have `chmod +x` run on it.

**Comment by aaronedell** [comment]
Unfortunately I think this is a known issue.

**Comment by sprak3000** [comment]
@aaronedell 

Forgot to mention... Please run `chmod +x` on the script file.

**Review by sprak3000 — APPROVED** [review]

**Pull request merged**